CHANGELOG.md

```diff
@@ -61,8 +61,13 @@ Please refer to the [Breaking changes](https://developers.arcgis.com/javascript/
 
 ## Bug fixes and enhancements
 
+- BUG-000156080: Fixed an issue where requests for features not visible or displayed would still fire when [snapping using the Editor](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-interactive-snapping-SnappingOptions.html).
 - [Esri Community - 1043965](https://community.esri.com/t5/arcgis-javascript-maps-sdk-questions/querying-geojson-failed-in-4-18/m-p/1043965): Fixed an issue where the [GeoJSONLayer.queryFeatures()](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-GeoJSONLayer.html#queryFeatures) not returning results when geometry's extent is used for query.
 - [Esri Community - 1307834](https://community.esri.com/t5/arcgis-javascript-maps-sdk-questions/attachments-widget-bug/m-p/1307834): Fixed an issue where the [Attachments](https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Attachments.html) widget throws an error saying `cannot set properties of null even though graphic valid`.
+- [Esri Community - 1308220](https://community.esri.com/t5/arcgis-javascript-maps-sdk-questions/searchwidget-with-view-no-layerview-for-layer/m-p/1308220): Fixed an issue with displaying an error message in the browser console when creating a [FeatureLayer](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-FeatureLayer.html) on-the-fly to use as a [LayerSearchSource](https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Search-LayerSearchSource.html) with the [Search](https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Search.html) widget.
+- Fixed an accessibility issue in the [Legend](https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Attachments.html) widget to avoid placing inactive elements in the focus order.
```

Comment:
is this link supposed to be to Legend instead of the Attachments widget?
```suggestion
- Fixed an accessibility issue in the [Legend](https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Legend.html) widget to avoid placing inactive elements in the focus order.
```

Reply:
Yes - good catch! Copy/pasted wrong 😅  thanks!